lib/defaults/config.json

```diff
@@ -35,7 +38,9 @@
     "urlEncoded": {
       "extended": true
     },
-    "json": {}
+    "json": {},
```

Comment:
The individual bodyParser params should be exempt, not the bodyParser object itself.

Reply:
ah right, good catch

Reply:
what troy is saying we is we want to require the `json` and `urlEncoded` params, but we don't care what's in them

Reply:
Ah alright I can do that